.github/workflows/lazy.yml

```diff
@@ -44,3 +44,35 @@ jobs:
         run: cargo build --release --features spv-in,msl-out --bin naga
       - name: Convert shaders
         run: for file in data/*.spv ; do echo "Translating" ${file} && target/release/naga --validate 27 ${file} ${file}.metal; done
+  parse-vulkan-tutorial-shaders:
+    name: Parse Sascha Willems Vulkan tutorial shaders
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+      - name: Download shaders
+        run: git clone https://github.com/SaschaWillems/Vulkan.git
+      - name: Build Naga
+        run: cargo build --release --features spv-in,msl-out --bin naga
+      - name: Convert metal shaders
+        run: |
+          # No needed to stop workflow if we can't validate one file
+          set +e
+          touch counter
+          SUCCESS_RESULT_COUNT=0
+          FILE_COUNT=0
+          mkdir -p out
+          find "Vulkan/data/shaders/glsl/" -name '*.spv' | while read fname;
+          do
+              echo "Convert: $fname"
+              FILE_COUNT=$((FILE_COUNT+1))
+              target/release/naga --validate 27 $(realpath ${fname}) out/$(basename ${fname}).metal
```

Comment:
Should we test any backend here? Or just test `spv-in` frontend? If we decide to tests the backend here we should also validate output shaders.

Reply:
We should probably enable the backends as they become operational